src/net_processing.cpp

```diff
@@ -1453,6 +1453,15 @@ bool static AlreadyHave(const CInv& inv) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
     return true;
 }
 
+void RelayTransaction(const uint256& txid, const CConnman& connman)
```

Comment:
16452: missing some previous PR

Reply:
Earliest records point to [`#9260`](https://github.com/bitcoin/bitcoin/pull/9260/files#diff-6875de769e90cec84d2e8a9c1b962cdbcda44d870d42e4215827e599e11e90e3R843-R850) so backporting is out of scope.